gdnative-core/src/geom/transform.rs

```diff
@@ -24,4 +26,22 @@ impl Transform {
     pub fn from_sys(c: sys::godot_transform) -> Self {
         unsafe { std::mem::transmute::<sys::godot_transform, Self>(c) }
     }
+
+    pub fn translate(origin: Vector3) -> Transform {
+        Transform {
+            basis: Basis::identity(),
+            origin,
+        }
+    }
+
+    pub fn from_transform<Src, Dst>(transform: &Transform3D<f32, Src, Dst>) -> Transform {
```

Comment:
If you added a untyped version of `Basis::from_transform`, add one for this as well.

Reply:
It could be nice to also add a `From<Transform3D<f32, Src, Dst>>` implementation on `Transform`, since this one covers all aspects of a transform, unlike `Basis::from_transform`.